tests/test_conversion_base.py

```diff
@@ -1866,6 +1866,79 @@ def test_convert_collect_error(test_backend):
     assert res == [] and error[0] == rule and isinstance(error[1], SigmaValueError)
 
 
+def test_convert_errors_class_variable_issue(test_backend):
+    test_backend.collect_errors = True
+
+    collection = SigmaCollection.from_yaml(
+        """
+            title: Test
+            status: test
+            logsource:
+                category: test_category
+                product: test_product
+            detection:
+                sel: true
+                condition: sel
+        """
+    )
+    rule = collection.rules[0]
+
+    res = test_backend.convert(collection)
+    error = test_backend.errors[0]
```

Comment:
The errors is populated in the first instance of the `TextQueryTestBackend`, but it is not shared with other instances, hence the following [assertion](https://github.com/SigmaHQ/pySigma/pull/145/files#diff-3f29f910daccbbc3b1a10df036c75fc2ce887d9b8bdddb03ee528f79aa322de0R1939) succeeds.